.github/workflows/pre-release.yml

```diff
@@ -55,7 +57,7 @@ jobs:
       - name: Push changes
         run: git push --follow-tags
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```

Comment:
is that something from testing out the release candidates? Or should this be checked in?

Reply:
This was failing for pre-lease so I figured it would fail here as well.